impeller/renderer/backend/vulkan/driver_info_vk_unittests.cc

```diff
@@ -4,7 +4,11 @@
 
 #include "impeller/playground/playground_test.h"
 #include "impeller/renderer/backend/vulkan/context_vk.h"
+#include "impeller/renderer/backend/vulkan/driver_info_vk.h"
 #include "impeller/renderer/backend/vulkan/surface_context_vk.h"
+#include "impeller/renderer/backend/vulkan/test/mock_vulkan.h"
+#include "third_party/swiftshader/include/vulkan/vulkan_core.h"
```

Comment:
Both this one and the next are bad includes. vk.h should suffice and sets up the right defines before import.

Perhaps the IDE is including it?

Reply:
Done